kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/menu/MenuUtilsTest.java

```diff
@@ -80,4 +84,17 @@ public void testBuildItemWithNormalIsWidget() {
         menuItem.setEnabled(false);
         assertFalse(menuItem.isEnabled());
     }
+
+    @Test
+    public void testBuildHasEnabledWidget() {
+        MenuUtils.HasEnabledIsWidget hasEnabledIsWidget = MenuUtils.buildHasEnabledWidget(button);
+
+        assertEquals(button, hasEnabledIsWidget.asWidget());
+        assertEquals(button.getTitle(), hasEnabledIsWidget.asWidget().getTitle());
+        assertEquals(false, hasEnabledIsWidget.isEnabled());
+
+        hasEnabledIsWidget.setEnabled(true);
+        when(hasEnabledIsWidget.isEnabled()).thenReturn(true);
```

Comment:
This mocking doesn't make sense (or the following assert is useless) - you're basically testing, that mockito mocked the call to `isEnabled()`correctly? But hasEnabledIsWidget isn't a mock, or is it?